apps/docs/reference/presence.md

```diff
@@ -1,68 +1,145 @@
+    state: videoElement.paused ? "Playback paused" : "Playing",
+  };
+
+  // use setActivity to update the user's activity
+  presence.setActivity(presenceData);
 });
 ```
 
-## Constructor
+## constructor(options: [`PresenceOptions`](#presence-options)) {#constructor}
+
+## setActivity(data: [`PresenceData`](#presence-data) | [`Slideshow`](#slideshow))
+
+The main way to update the user's activity.
 
-### `new Presence(options: PresenceOptions)`
+It can also be used with Slideshows, read more at [`createSlideshow`](#createSlideshow)
 
-Creates a new `Presence` instance.
+```ts{16}
+const presenceData: PresenceData = {
+	name: "Youtube",
+	type: ActivityType.Watching,
+	details: "The Best Speedtech Cup of The Day in 2024!",
+	state: "WirtualTV",
+	largeImageKey: "https://img.youtube.com/vi/gQ3A9Ph1Nuo/maxresdefault.jpg",
+	largeImageText: "Watching a video by WirtualTV",
+	smallImageKey: "https://i.imgur.com/6yp7YeO.jpeg",
+	smallImageText: "WirtualTV",
+	buttons: [
+		{ label: "Watch", url: "https://youtu.be/gQ3A9Ph1Nuo" },
+	],
+};
 
-#### Parameters
+// Update the user's activity
+presence.setActivity(presenceData);
+```
+
+::: details Example with Slideshow
 
-- `options` (`PresenceOptions`): The options for the Presence.
+```ts
+// Creates a new slideshow
+const slideshow = presence.createSlideshow();
 
-#### Returns
+// Defines two activities for the slideshow
+const likesPresence = { details: "Video has 1.2k likes!" };
+const viewsPresence = { details: "Video has 80k views!" };
 
-- `Presence`: The new `Presence` instance.
+// Add the two activities with 5 seconds intervals
+slideshow.addSlide("likeCounter", likesPresence, 5000);
+slideshow.addSlide("viewsCounter", viewsPresence, 5000);
 
-## Properties
+// Update the user's activity, toggling between the two every 5 seconds
+presence.setActivity(slideshow);
+```
 
-### `clientId: string`
+For more details check [Slideshow](#TODO)
+:::
 
-The Client ID of the Presence.
+## getStrings(strings: Record\<string,string\>)
 
-## Methods
+Used to fetch translations from the extension.
 
-### `setActivity(activity: PresenceActivity)`
+a list of translations can be found in the [Translations Repository](#TODO)
 
-Sets the activity of the Presence.
+to add your own custom translations check [Custom Translations](#TODO)
 
-#### Parameters
+```ts
+// type definition
+function getStrings<T extends Record<string, string>>(strings: T): Promise<T>;
+```
+
+Example
+
+```ts {7-9}
+const strings = await presence.getStrings({
+  playing: "general.playing",
+  paused: "general.paused",
+  search: "general.search",
+}); // { playing: "Playing", paused: "Paused", search: "Searching" }
+
+console.log(strings.paused); // EN: Paused
+console.log(strings.playing); // DE: Spielt
+console.log(strings.search); // RU: Ищет
+```
 
-- `activity` (`PresenceActivity`): The activity to set.
+## clearActivity()
+
+Clears the user's activity, can be used when the user is idle (e,g: video paused)
+
+Example
+
+```ts{4-5}
```

Comment:
```suggestion
```ts
```

Reply:
These numbers are used for line highlights
![image](https://github.com/user-attachments/assets/d9fe6679-a901-4605-b8b1-9e6bd08603bb)

Reply:
![image](https://github.com/user-attachments/assets/f54d86df-e3ff-4ceb-98ad-4c3de47af250)

Reply:
this is an extended markdown file, to preview check the deployed build in inital comment